ssl/statem/extensions.c

```diff
@@ -653,6 +680,13 @@ int tls_collect_extensions(SSL_CONNECTION *s, PACKET *packet,
             SSLfatal(s, SSL_AD_ILLEGAL_PARAMETER, SSL_R_BAD_EXTENSION);
             goto err;
         }
+#ifndef OPENSSL_NO_ECH
```

Comment:
I think I convinced myself at some stage there may be a crash there and added the comment to remind me to check with some unsupported extension. For the moment, I've added a TODO to make such a test - could be I was wrong in the first place, or something's been fixed since.